site/src/components/LicenseCard/LicenseCard.tsx

```diff
@@ -72,7 +75,7 @@ export const LicenseCard = ({
           <Stack direction="column" spacing={0} alignItems="center">
             <span className={styles.secondaryMaincolor}>Users</span>
             <span className={styles.userLimit}>
-              {userLimitActual} {` / ${userLimitLimit || "Unlimited"}`}
+              {userLimitActual} {` / ${currentUserLimit || "Unlimited"}`}
```

Comment:
Should we move the condition `|| "Unlimited"` to `currentUserLimit`?

Reply:
I personally would rather not doing it since the user limit is a number and here we are dealing with "presentational" logic

Reply:
Killer argument 👍